src/lib/Libifl/pbs_loadconf.c

```diff
@@ -1188,6 +1189,107 @@ __pbs_loadconf(int reload)
 	return 0;
 }
 
+/**
+ * @brief
+ *	convert psi to comma separated string.
+ *
+ * @param[in] psi - psi structure
+ * @param[in] nsvrs - num of servers
+ * 
+ * @note
+ * return string has to be freed by the caller
+ *
+ * @return char *
+ * @retval !NULL pointer psi string
+ * @retval NULL failure
+ */
+char *
+psi_to_str(psi_t *psi, int nsvrs)
```

Comment:
Do we need nsvrs as a separate argument? We can just call get_num_servers() right.